cross-platform/react-app/src/frontend/Screens/Hub/HubScreen.tsx

```diff
@@ -49,11 +49,17 @@ export interface HubScreenProps {
   onOpen: (filename: string, iModelPromise: Promise<IModelConnection>) => Promise<void>;
   /** Callback called when the back button is pressed to go to the previous screen. */
   onBack: () => void;
+  /**
+   * If the performActions includes and OPEN action with a remote: prefix, this will be set to
```

Comment:
I think this should be: includes **an** OPEN action

Reply:
Fixed.